arkouda/strings.py

```diff
@@ -525,6 +525,50 @@ def is_title(self) -> pdarray:
             generic_msg(cmd="checkChars", args=f"isTitle {self.objtype} {self.entry.name}")
         )
 
+    @typechecked
+    def strip(self, pattern: Union[bytes, str_scalars] = " ") -> Strings:
+        """
+        Returns a new Strings with all leading and trailing spaces (or set of given characters)
+        stripped from the original.
+
+        Returns
+        -------
+        Strings
+            Strings with all leading and trailing spaces (or set of given characters) stripped from
+            the original
+
+        Raises
+        ------
+        RuntimeError
+            Raised if there is a server-side error thrown
+
+        See Also
+        --------
+        Strings.to_upper
```

Comment:
I am not sure to_upper needs to be referenced here.

Reply:
Yep. Cut and paste leftover.